controllers/tab/settings/appearance/form/PKPAppearanceForm.inc.php

```diff
@@ -39,6 +39,11 @@ function PKPAppearanceForm($wizardMode = false, $additionalSettings = array()) {
 
 		AppLocale::requireComponents(LOCALE_COMPONENT_APP_COMMON);
 
+		$themes = PluginRegistry::getPlugins('themes');
```

Comment:
This is where permitting plugins to cleanly double-load might be a good idea. If the `PKPTemplateManager` call to `loadCategory('themes')` only loads enabled plugins (advisable for speed), then this one will be needed to load the others that aren't enabled, and as a consequence some plugins (the enabled ones) may be loaded twice. They'll therefore need to ensure that double-registration for hooks either doesn't happen or is benign.